Dockerfile

```
@@ -1,21 +1,38 @@
+COPY --from=ghcr.io/astral-sh/uv:0.6 /uv /uvx /bin/
 
-RUN python3 -m venv /opt/acoustid/server.venv && \
-    /opt/acoustid/server.venv/bin/pip install --no-cache-dir -r /tmp/requirements.txt
+RUN useradd -ms /bin/bash acoustid
+WORKDIR /opt/acoustid/server
 
-WORKDIR /opt/acoustid/server/
+FROM base AS builder
 
-COPY ./ /opt/acoustid/server/
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)